client/components/Reports/SummarizeTestPlanReport.jsx

```diff
@@ -95,7 +95,8 @@ const getTestersRunHistory = (
         if (
             testPlanReportId === testPlanReport.id &&
             testPlanReport.status === 'FINALIZED' &&
-            testResult
+            testResult &&
+            testResult.completedAt
```

Comment:
It could be `testResult?.completedAt` which is more compact 🙂 but that's optional because adding a second line might make the intent more clear.

Reply:
I'm okay with making this change, thanks